pkg/plugin/v3/webhook.go

```diff
@@ -96,6 +100,19 @@ func (p *createWebhookPlugin) Validate() error {
 			" --programmatic-validation and --conversion to be true", p.commandName)
 	}
 
+	if p.conversion {
+		if (p.spoke == "" && p.hub != "") || (p.hub == "" && p.spoke != "") {
+			return fmt.Errorf("Hub and Spoke version need to be specified when using conversion webhooks")
+		}
+		if p.spoke == "" || p.hub == "" {
+			fmt.Println("Hub and Spoke version can be specified in the command to scaffold Convertible and Hub interfaces")
```

Comment:
Is it a warn/info? I do not think that it is required,

Reply:
This is an additional info informing users that the feature is available for them to set.

Reply:
IMO we ough to not use the `fmt.Println` to print WARN/INFO logs for the tool. We can use either logr with zapr, or klog v2